app/src/main/kotlin/com/intive/picover/common/validator/TextValidator.kt

```diff
@@ -19,13 +19,16 @@ class TextValidator private constructor(
 	}
 
 	class Builder {
-		private var allowEmpty = true
-		private var allowBlank = true
-		private var maxLength: Int? = null
+		var allowEmpty = true
+		var allowBlank = true
+		var maxLength: Int? = null
```

Comment:
getter should be private

Reply:
So the property needs to be private and a custom setter needs to be added as it was before. Why in your opinion the getter need to be private?

Reply:
Oh right, we have a limitation here. 
Why getter should be private? Because this is how the builder design pattern looks like :) 
Every build has only setters and `build` function and accessing e.g. `builder.name` shouldn't be allowed. 
So let's keep it as it is.